gcp_bootstrap/dataproc_init.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# To be copied to gs://moz-fx-data-prod-airflow-artifacts
+
+set -exo pipefail
+
+# Logs will be available on the dataproc nodes at /var/log/dataproc-initialization-script-X.log
+# or via the GCP Dataproc UI
+
+ARTIFACTS_BUCKET=gs://moz-fx-data-prod-airflow-artifacts
+
+ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+if [[ "${ROLE}" == 'Master' ]]; then
+  # You can put any master-specific logic here
+  echo "Running dataproc_init.sh on master..."
+fi
+
+gsutil cp $ARTIFACTS_BUCKET/jars/* /usr/lib/spark/jars/
+
+# Install so psycopg2 requirement for python_mozaggregator doesn't complain
```

Comment:
Between this and the `requirements.txt` install later, I think it might be best to let each job define its own install environment (but if we do that, we might as well dockerize all of this and run it on GKE...)